PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/Endpoint/endpoint_app.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+#include "./controller_class/configuration_controller.h"
+#include "memory_map.h"
+#include "../utils/tlp.h"
+#include "memory_controller.h"
+
+class EndpointApp {
+public:
+    EndpointApp(uint64_t hostMemorySize, uint64_t deviceMemorySize);
```

Comment:
I can't understand the purpose of these two arguments, our memories sizes and types are known before (they have given it to us in the specifications of the endpoint)